frontend/src/App.tsx

```
@@ -1,62 +1,80 @@
+const About = lazy(() => import("./pages/About"));
+const Documentation = lazy(() => import("./pages/Documentation"));
+const FAQ = lazy(() => import("./pages/FAQ"));
+const Onboarding = lazy(() => import("./pages/Onboarding"));
+const NotFound = lazy(() => import("./pages/NotFound"));
 
 const queryClient = new QueryClient();
 
+function PageLoader() {
+  const { t } = useTranslation();
+  return (
+    <main id="main-content" tabIndex={-1} className="min-h-screen flex items-center justify-center" role="status">
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)